src/geometry/Lie_group/lie_group.lean

```diff
@@ -0,0 +1,258 @@
+/-
```

Comment:
@Nicknamen, one outstanding request from Johan above was to rename the `Lie_group` folder to `lie_group`, for consistency.

Reply:
Indeed I did replace everything, did I not? Please let me know which ones I did not replace

Reply:
I thought his request was to replace the instances of `Lie_group` with `lie_group`, I did not understand he was referring specifically to the folder. I will change it, but in any case I will change the name of the folder to algebra at some point, when I will implement other smooth structures.

Reply:
I was actually also referring to the names in the code. We have a definition `lie_algebra`. Hence I would also like to have `lie_group`.

Reply:
Ooh, I should learn to read the diffs before replying... :facepalm: